core/src/main/scala/sttp/tapir/Codec.scala

```diff
@@ -91,7 +93,50 @@ object Codec extends MultipartCodecDerivation with FormCodecDerivation {
 
   def stringCodec(charset: Charset): PlainCodec[String] = plainCodec(identity, charset)
 
-  private def plainCodec[T: Schema](parse: String => T, charset: Charset = StandardCharsets.UTF_8): PlainCodec[T] =
+  implicit val localTimePlainCodec: PlainCodec[LocalTime] = plainCodec[LocalTime](LocalTime.parse)
+  implicit val localDatePlainCodec: PlainCodec[LocalDate] = plainCodec[LocalDate](LocalDate.parse)
+
+  implicit val zonedDateTimePlainCodec: PlainCodec[ZonedDateTime] =
+    new PlainCodec[ZonedDateTime] {
+      val formatter: DateTimeFormatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME
+      val charset: Charset = StandardCharsets.UTF_8
+
+      override def encode(t: ZonedDateTime): String = t.format(formatter)
+
+      override def rawDecode(s: String): DecodeResult[ZonedDateTime] = {
+        try {
+          Value(ZonedDateTime.parse(s, formatter))
+        } catch {
+          case e: Exception => Error(s, e)
+        }
+      }
+
+      override val meta: CodecMeta[ZonedDateTime, CodecFormat.TextPlain, String] =
+        CodecMeta(implicitly, CodecFormat.TextPlain(charset), StringValueType(charset))
+    }
+
+  implicit val localDateTimeCodec: PlainCodec[LocalDateTime] =
```

Comment:
shouldn't we change the schema for `LocalDateTime` to `DateTime`? It's `String` currently

Reply:
As far as I can see in `Schema`, it's `date-time`:
```scala
  implicit val schemaForLocalDateTime: Schema[LocalDateTime] = Schema(SDateTime)
```

Reply:
Ah yes 🤦‍♂ I looked at `LocalTime` sorry